package.json

```diff
@@ -9,14 +9,11 @@
   },
   "engines": {
     "node": ">=16",
-    "pnpm": ">=7.24.2 <7.33.0"
-  },
-  "engines:comment": {
-    "pnpm": "Our lock file version is 6. PNPM 7.24.2 is the first PNPM version to support version 6 lock files. PNPM 7.33.0 adds support for version 6.1 lock files, so using it results in a `pnpm-lock.yaml` diff."
+    "pnpm": ">=8"
   },
   "volta": {
     "node": "18.17.0",
-    "pnpm": "7.32.5"
+    "pnpm": "8.6.11"
```

Comment:
@ynotdraw: I saw some discussion elsewhere about needing to upgrade Embroider to unlock the PNPM 8 upgrade. Am I missing something?

Reply:
Not that I recall!  If the build and local dev is happy, I'm happy!